DESIGN.md

```diff
@@ -561,9 +561,3 @@ CONNECT /sessionPlayers?connection=$connectionToken&session=$sessionToken&player
 
 This connection may be truly peer-to-peer but if reachability is an issue it may be tunneled through a STUN or TURN server. An implementation detail the client does not need to worry about.
 
-
```

Comment:
Pity there is no way to show markdeep in GitHub

Reply:
Yes, 100%. I'm currently treating the markdeep header/footer as something that happens in a publish step. Those lines are wreaking havoc in all the wysiwyg markdown editors I use.